From: Ross Vandegrift <ross@kallisti.us>
Date: Sat, 5 Apr 2014 17:33:27 -0400
Subject: Fix minor issues found by pyflakes

Two variable references are class instance properties.
python3 removes cmp, but we have __eq__ anyway.

Signed-off-by: Ross Vandegrift <ross@kallisti.us>
---
 src/rawapi.py | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/src/rawapi.py b/src/rawapi.py
index f4fa34d..e267848 100644
--- a/src/rawapi.py
+++ b/src/rawapi.py
@@ -58,9 +58,6 @@ class SaneEnum(object):
             return self.__value == other
         return self.__value == other.__value
 
-    def __cmp__(self, other):
-        return cmp(self.__value, other.__value)
-
     def __str__(self):
         if not self.__value in self.VALUE_TO_STR:
             txt = "Unknown value (%d)" % (self.__value)
@@ -422,9 +419,9 @@ class __AuthCallbackWrapper(object):
         (login, password) = self.__auth_callback(ressource_ptr.value)
         # TODO(Jflesch): Make sure the following works
         ctypes.memmove(login_ptr, ctypes.c_char_p(login),
-                       min(len(login)+1, MAX_USERNAME_LEN))
+                       min(len(login)+1, self.MAX_USERNAME_LEN))
         ctypes.memmove(passwd_ptr, ctypes.c_char_p(password),
-                       min(len(password)+1, MAX_USERNAME_LEN))
+                       min(len(password)+1, self.MAX_USERNAME_LEN))
 
 
 AUTH_CALLBACK_DEF = ctypes.CFUNCTYPE(None, ctypes.c_char_p, ctypes.c_char_p,
